lib/expansion_rules.rb

```diff
@@ -65,9 +65,9 @@ def details_fields(*fields)
   STEP_BY_STEP_FIELDS = (DEFAULT_FIELDS + [%i(details step_by_step_nav title), %i(details step_by_step_nav steps)]).freeze
   TRAVEL_ADVICE_FIELDS = (DEFAULT_FIELDS + details_fields(:country, :change_description)).freeze
   WORLD_LOCATION_FIELDS = [:content_id, :title, :schema_name, :locale, :analytics_identifier].freeze
-  FACET_GROUP_FIELDS = (%i[content_id title schema_name] + details_fields(:name, :description)).freeze
+  FACET_GROUP_FIELDS = (%i[content_id title locale schema_name] + details_fields(:name, :description)).freeze
```

Comment:
Is this a bug fix?

Reply:
Yeah, this is necessary to comply with the notification schema links definition.

Reply:
Ah. Could you add that to the commit note? I don't think locale is required in the [links schema](https://github.com/alphagov/govuk-content-schemas/blob/master/dist/formats/facet_group/publisher_v2/links.json).

Reply:
It's not required in the publisher_v2 links schema, it's required in the [frontend links schema](https://github.com/alphagov/govuk-content-schemas/blob/master/formats/shared/definitions/frontend_links.jsonnet#L9) I think the downstream payloads are validated against this.